docs/source/models/risk.rst

```diff
@@ -4,17 +4,100 @@
 Modeling Risk Exposure
 ======================
 
-.. todo::
+.. note::
 
-   Overview of modeling GBD risk factor exposure.
+   Vivarium has the infrastructure to model any risks in the GBD hierarchy. 
+   You should use tools from ``vivarium_inputs`` to pull and examine the 
+   data for your risks before including them in your model.
 
-.. contents:
+GBD Overview
+------------
 
 What is a risk factor?
-----------------------
+++++++++++++++++++++++
+A risk factor is any attribute, characteristic or exposure of an individual 
+that increases the likelihood of developing a disease or injury. Some examples 
+of the more important risk factors are underweight, unsafe sex, high blood pressure, tobacco and alcohol consumption, sanitation and hygiene, and unsafe water. [WHO]_ In GBD, a risk factor refers to any exposure that leads to a 
+loss of health in the population; it can be anywhere on the causal chain as 
+long as there is evidence to show that the risk factor is causally linked to an outcome disease or injury.
+
+What do we measure?
++++++++++++++++++++
+- Proportion of disease and injury burden attributable to specific risk factors
+   - e.g. burden of diarrheal due to unsafe water
+   - e.g. burden of lung cancer due to smoking
+- Comparative importance of the underlying risk factors which cause disease and
+  injuries
+   - e.g. burden of smoking compared with unsafe water
```

Comment:
I would specifically name 'population attributable fraction' and 'summary exposure value' as the terms for these definitions here.